assets/js/components/adminbar/AdminBarApp.js

```diff
@@ -25,20 +25,25 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies.
  */
+import AdminBarUniqueVisitors from './AdminBarUniqueVisitors';
+import AdminBarSessions from './AdminBarSessions';
+import AdminBarImpressions from './AdminBarImpressions';
+import AdminBarClicks from './AdminBarClicks';
+import AdminbarModules from './LegacyAdminBarModules';
+import AnalyticsInactiveCTA from '../AnalyticsInactiveCTA';
 import Data from 'googlesitekit-data';
-import {
-	decodeHTMLEntity,
-	trackEvent,
-} from '../../util';
-import AdminbarModules from './adminbar-modules';
 import Link from '../Link';
 import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import { decodeHTMLEntity, trackEvent } from '../../util';
 const { useSelect } = Data;
 
 export default function AdminBarApp() {
 	const currentEntityURL = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
 	const currentEntityTitle = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityTitle() );
 	const detailsURL = useSelect( ( select ) => select( CORE_SITE ).getAdminURL( 'googlesitekit-dashboard', { permaLink: currentEntityURL } ) );
+	const analyticsModuleActive = useSelect( ( select ) => select( CORE_MODULES ).isModuleActive( 'analytics' ) );
```

Comment:
We shouldn't need both of these selected since a module cannot be connected when inactive (but the inverse is possible).

```suggestion
```

Reply:
Updated @aaemnnosttv